6001-7000/LIT6936TarWaAmidBeta.xml

```diff
@@ -161,28 +162,28 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             </div>
             <div type="edition">
                 <note>This division into textparts and labels follows the devision in <bibl><ptr target="bm:Diez2024Amid"/></bibl>.</note> 
-                <div type="textpart" subtype="part" xml:id="Preface">
+                <div type="textpart" subtype="part" xml:id="Preface" corresp="#J #M">
                     <label>Attribution to the author by the translator.</label>
-                    <note>The following text is taken from <ref target="#J #M"/>.</note>
+                    <note>The following text is taken from <ref type="mss" corresp="FSUor134"/> and <ref type="mss" corresp="BLorient814"/>.</note>
```

Comment:
in this context you can of course refer to the xml:id of the record, but this is not acceptable in a real critical edition